inst/pages/30_differential_abundance.qmd

```diff
@@ -8,8 +8,7 @@ chapterPreamble()
 This section provides an introduction to Differential Abundance Analysis (DAA),
 which is used to identify differences in the abundances of individual taxa (at
 any taxonomic level) between two or more groups, such as treatment and control.
-Here, we demonstrate its implementation on Tengeler2020, described in chapter
-[@sec-tengeler-desc].
+Here, we demonstrate its implementation on Tengeler2020, described in [this reference](https://microbiome.github.io/mia/reference/Tengeler2020.html).
```

Comment:
Keep it short, like

Here, we demonstrate its implementation on [Tengeler2020](https://microbiome.github.io/mia/reference/Tengeler2020.html) data set.